### PR TITLE
Update documentation of KQL query search and conditions

### PR DESCRIPTION
**Pull request by CoreySchneider-MSFT**
Added link to "recipient expansion" section for relevant fields.

### CODE REVIEW

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 1]
```diff
@@ -53,19 +53,19 @@ The following table lists email message properties that can be searched by using
 |Property|Property description|Examples|Search results returned by the examples|
 |---|---|---|---|
 |AttachmentNames|The names of files attached to an email message.|`attachmentnames:annualreport.ppt` <p> `attachmentnames:annual*` <br/> `attachmentnames:.pptx`|Messages that have an attached file named annualreport.ppt. In the second example, using the wildcard character ( * ) returns messages with the word "annual" in the file name of an attachment. The third example returns all attachments with the pptx file extension.|
-|Bcc|The Bcc field of an email message.<sup>1</sup>|`bcc:pilarp@contoso.com` <p> `bcc:pilarp` <p> `bcc:"Pilar Pinilla"`|All examples return messages with Pilar Pinilla included in the Bcc field.|
+|Bcc|The Bcc field of an email message.<sup>1</sup>|`bcc:pilarp@contoso.com` <p> `bcc:pilarp` <p> `bcc:"Pilar Pinilla"`|All examples return messages with Pilar Pinilla included in the Bcc field.<br>([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:56]:
```suggestion
|Bcc|The Bcc field of an email message.<sup>1</sup>|`bcc:pilarp@contoso.com` <p> `bcc:pilarp` <p> `bcc:"Pilar Pinilla"`|All examples return messages with Pilar Pinilla included in the Bcc field.<br>(See [Recipient expansion](recipient-expansion.md).)|
```

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 2]
```diff
@@ -53,19 +53,19 @@ The following table lists email message properties that can be searched by using
...
 |Category|The categories to search. Categories can be defined by users by using Outlook or Outlook on the web (formerly known as Outlook Web App). The possible values are: <ul><li>blue<li>green<li>orange<li>purple<li>red<li>yellow</li></ul>|`category:"Red Category"`|Messages that have been assigned the red category in the source mailboxes.|
-|Cc|The Cc field of an email message.<sup>1</sup>|`cc:pilarp@contoso.com` <p> `cc:"Pilar Pinilla"`|In both examples, messages with Pilar Pinilla specified in the Cc field.|
+|Cc|The Cc field of an email message.<sup>1</sup>|`cc:pilarp@contoso.com` <p> `cc:"Pilar Pinilla"`|In both examples, messages with Pilar Pinilla specified in the Cc field.<br>([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:58]:
```suggestion
|Cc|The Cc field of an email message.<sup>1</sup>|`cc:pilarp@contoso.com` <p> `cc:"Pilar Pinilla"`|In both examples, messages with Pilar Pinilla specified in the Cc field.<br>(See [Recipient expansion](recipient-expansion.md).)|
```

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 3]
```diff
@@ -53,19 +53,19 @@ The following table lists email message properties that can be searched by using
...
-|From|The sender of an email message.<sup>1</sup>|`from:pilarp@contoso.com` <p> `from:contoso.com`|Messages sent by the specified user or sent from a specified domain.|
+|From|The sender of an email message.<sup>1</sup>|`from:pilarp@contoso.com` <p> `from:contoso.com`|Messages sent by the specified user or sent from a specified domain.<br>([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:60]:
```suggestion
|From|The sender of an email message.<sup>1</sup>|`from:pilarp@contoso.com` <p> `from:contoso.com`|Messages sent by the specified user or sent from a specified domain.<br>(See [Recipient expansion](recipient-expansion.md).)|
```

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 4]
```diff
@@ -53,19 +53,19 @@ The following table lists email message properties that can be searched by using
...
-|Participants|All the people fields in an email message. These fields are From, To, Cc, and Bcc.<sup>1</sup>|`participants:garthf@contoso.com` <p> `participants:contoso.com`|Messages sent by or sent to garthf@contoso.com. The second example returns all messages sent by or sent to a user in the contoso.com domain.|
+|Participants|All the people fields in an email message. These fields are From, To, Cc, and Bcc.<sup>1</sup>|`participants:garthf@contoso.com` <p> `participants:contoso.com`|Messages sent by or sent to garthf@contoso.com. The second example returns all messages sent by or sent to a user in the contoso.com domain.<br>([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:66]:
```suggestion
|Participants|All the people fields in an email message. These fields are From, To, Cc, and Bcc.<sup>1</sup>|`participants:garthf@contoso.com` <p> `participants:contoso.com`|Messages sent by or sent to garthf@contoso.com. The second example returns all messages sent by or sent to a user in the contoso.com domain.<br>(See [Recipient expansion](recipient-expansion.md).)|
```

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 5]
```diff
@@ -53,19 +53,19 @@ The following table lists email message properties that can be searched by using
...
 |Received|The date that an email message was received by a recipient.|`received:2021-04-15` <p> `received>=2021-01-01 AND received<=2021-03-31`|Messages that were received on April 15, 2021. The second example returns all messages received between January 1, 2021 and March 31, 2021.|
-|Recipients|All recipient fields in an email message. These fields are To, Cc, and Bcc.<sup>1</sup>|`recipients:garthf@contoso.com` <p> `recipients:contoso.com`|Messages sent to garthf@contoso.com. The second example returns messages sent to any recipient in the contoso.com domain.|
+|Recipients|All recipient fields in an email message. These fields are To, Cc, and Bcc.<sup>1</sup>|`recipients:garthf@contoso.com` <p> `recipients:contoso.com`|Messages sent to garthf@contoso.com. The second example returns messages sent to any recipient in the contoso.com domain.<br>([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:68]:
```suggestion
|Recipients|All recipient fields in an email message. These fields are To, Cc, and Bcc.<sup>1</sup>|`recipients:garthf@contoso.com` <p> `recipients:contoso.com`|Messages sent to garthf@contoso.com. The second example returns messages sent to any recipient in the contoso.com domain.<br>(See [Recipient expansion](recipient-expansion.md).)|
```

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 6]
```diff
@@ -208,7 +208,7 @@ Create a condition using common properties when searching mailboxes and sites in
 |Condition|Description|
 |---|---|
 |Date|For email, the date a message was received by a recipient or sent by the sender. For documents, the date a document was last modified.|
-|Sender/Author|For email, the person who sent a message. For documents, the person cited in the author field from Office documents. You can type more than one name, separated by commas. Two or more values are logically connected by the **OR** operator.|
+|Sender/Author|For email, the person who sent a message. For documents, the person cited in the author field from Office documents. You can type more than one name, separated by commas. Two or more values are logically connected by the **OR** operator.<br>([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:211]:
```suggestion
|Sender/Author|For email, the person who sent a message. For documents, the person cited in the author field from Office documents. You can type more than one name, separated by commas. Two or more values are logically connected by the **OR** operator.<br>(See [Recipient expansion](recipient-expansion.md).)|
```

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 7]
```diff
@@ -220,10 +220,10 @@ Create a condition using mail properties when searching mailboxes or public fold
 |Condition|Description|
 |---|---|
 |Message kind|The message type to search. This is the same property as the Kind email property. Possible values: <ul><li>contacts</li><li>docs</li><li>email</li><li>externaldata</li><li>fax</li><li>im</li><li>journals</li><li>meetings</li><li>microsoftteams</li><li>notes</li><li>posts</li><li>rssfeeds</li><li>tasks</li><li>voicemail</li></ul>|
-|Participants|All the people fields in an email message. These fields are From, To, Cc, and Bcc.|
+|Participants|All the people fields in an email message. These fields are From, To, Cc, and Bcc. ([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:223]:
```suggestion
|Participants|All the people fields in an email message. These fields are From, To, Cc, and Bcc. (See [Recipient expansion](recipient-expansion.md).)|
```

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 8]
```diff
@@ -220,10 +220,10 @@ Create a condition using mail properties when searching mailboxes or public fold
...
 |Type|The message class property for an email item. This is the same property as the ItemClass email property. It's also a multi-value condition. So to select multiple message classes, hold the **CTRL** key and then click two or more message classes in the drop-down list that you want to add to the condition. Each message class that you select in the list will be logically connected by the **OR** operator in the corresponding search query. <p> For a list of the message classes (and their corresponding message class ID) that are used by Exchange and that you can select in the **Message class** list, see [Item Types and Message Classes](/office/vba/outlook/Concepts/Forms/item-types-and-message-classes).|
 |Received|The date that an email message was received by a recipient. This is the same property as the Received email property.|
-|Recipients|All recipient fields in an email message. These fields are To, Cc, and Bcc.|
+|Recipients|All recipient fields in an email message. These fields are To, Cc, and Bcc. ([See Recipient Expansion](https://docs.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide#recipient-expansionSee))|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:226]:
```suggestion
|Recipients|All recipient fields in an email message. These fields are To, Cc, and Bcc. (See [Recipient expansion](recipient-expansion.md).)|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CoreySchneider-MSFT, kindly commit the suggested changes. Many thanks.

**Review by jamesreedMSFT — APPROVED** [review]
These changes look good.

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**